config/locales/una_chapters_and_book_section.en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  hyrax:
+    icons:
+      una_chapters_and_book_section:     'fa fa-file-text-o'
```

Comment:
Why is there so many spaces before the string? If you removed the spaces, you should also remove the quotes. 

Also, you have a combination of single and double quotes.

Reply:
The codebase is a work in progress, so it doesn't need to be added for them all, but we can make the changes in this PR so that for next time its not left again. :)